IPFS-Practical.md

```diff
@@ -401,7 +401,7 @@ import { Contract, providers, utils } from "ethers";
 import Head from "next/head";
 import React, { useEffect, useRef, useState } from "react";
 import Web3Modal from "web3modal";
-import { abi, NFT_CONTRACT_ADDRESS } from "../constants";
+import { abi, NFT_CONTRACT_ADDRESS } from "../../constants";
```

Comment:
This should be `../constants` because the folder should exist outside of `pages`.

![Screenshot 2023-05-05 at 12 14 40 PM](https://user-images.githubusercontent.com/61384503/236392713-c4c8b329-0bd7-4333-9240-a2cacc672a68.png)

Reply:
Could you please clarify the comment. It should be ../../constants right?

Reply:
Hey @nithincp, sorry for the delay. 

No, it should be `../constants`. It seems you might have made a mistake in your part when following this code along.

Reply:
Okay I will check into that. Thanks for that.
Also what to do do with this PR now. Can you please guide me.
D I have to raise a new PR?

Reply:
Regarding the ../constants, 
For me when I used the create-next-app and created the folders, the pages folder was inside the src folder.
Hence I had to use the ../../constants.

I am attaching the screenshot
![image](https://github.com/LearnWeb3DAO/Junior-Track/assets/14857229/4c45d951-0d97-4591-9b4d-5b80c88bf0f2)

Reply:
I had given Yes when it asked if src directory has to be created(By default it was highlighted on Yes). Maybe thats why my directory structure got changed.